README.md

```diff
@@ -59,6 +59,21 @@ func main() {
 }
 ```
 
+### Custom aggregation for reported errors
+
+By default errors are aggregated by their _stack trace_ and _error message_. This might cause that errors that apparently are the same are treated as different in Periskop.
+
+To avoid that, you can manually group errors specifying the error key that you want to use:
```

Comment:
I'm confused here though. Where are we specifying the error key in the code below?

Reply:
Just read the implementation and realized that `json-parser-error` is the key.

Does this mean that it will apply the passed in key for grouping those and all the others won't be aggregated (unless the same error and stack appear?)

Reply:
This is used to bypass default aggregation method (which is stacktrace+error message). So, every time that you specify a key while reporting an error, all the errors are going to be aggregated with that key.